### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by java-live** [revelc/formatter-maven-plugin#469] [Q&A] [answered]
Hi there,

Wanted to know if there is a planned date for the next release. Due on june 30th ?

Regards

**Comment by hazendaz** [comment, marked as answer]
Generally after Eclipse releases quarterly updates which are currently expected around 3rd week of June.  Other releaeses will occur if code changes are needed.  I looked at what is currently merged, there are no pending code changes currently that I see other than third party library updates.  Is there something you are waiting on specifically there?

**Reply by java-live** [reply]
We have actually submitted some enhacements last month  that have been merged to the master (see pull request #457) . 
It basically helps to extend configuration where you would be able to use the formatter outside the project.
So my understanding is that unless it is released we wont be able to pull it from maven central.
We are now adopting this tool as primary formatter  for our java code base.

**Reply by hazendaz** [reply]
Ah, thanks I didn't see that when I first looked.  I'd suspect still end of June release as that change is so small and we know we are getting Eclipse updates then but will defer to @ctubbsii

**Reply by ctubbsii** [reply]
I was thinking this could be rolled into the next release, because it's so minor. However, if it's urgent, we can release it as 2.15.1

**Reply by java-live** [reply]
I think  it is more relvant to wait for the next release as it would contain more libs bumps and Eclipse major updates. 
All right thank you .
